app/frontend/stylesheets/config/_fonts.scss

```diff
@@ -0,0 +1,18 @@
+@import url('https://fonts.googleapis.com/css2?family=Special+Elite&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto&family=Special+Elite&display=swap');
+
+
+// Define fonts for body and headers
+$body-font: 'Roboto', sans-serif;
+$headers-font: 'Special Elite', cursive;
+
+
+// To use a font file (.woff) uncomment following lines
+// @font-face {
+//   font-family: "Font Name";
+//   src: font-url('FontFile.eot');
+//   src: font-url('FontFile.eot?#iefix') format('embedded-opentype'),
+//        font-url('FontFile.woff') format('woff'),
+//        font-url('FontFile.ttf') format('truetype')
+// }
+// $my-font: "Font Name";
```

Comment:
Fonts have already been imported inside the config/_variables.scss and the $body-font and $headers-font variables have already been defined so you can delete this file.

I think the fonts are not the same as the one you chose but they're the ones approved by Chloe, if we realize they don't fit as well as we'd like we can change them back later by changing the google import inside the config/_variables.scss file